src/types/index.ts

```diff
@@ -289,28 +289,6 @@ export type TxFunctionParsed = keyof typeof TX_FUNCTIONS;
 
 export type TxFunction = TxFunctionRaw | TxFunctionParsed;
 
-export type TxType =
```

Comment:
Instead of removing this I suggest to set the value of `keyof typeof Tag` to allow of reusing it within the app.

Reply:
Ideally `TxType` shouldn't be used. Only `Tag` must be used for the transaction type.

Reply:
OK